packages/notification-center/src/components/notification-center/components/notification-item/NotificationListItem.tsx

```diff
@@ -186,7 +186,7 @@ export function NotificationListItem({
                 css(listItemTimestampStyles)
               )}
             >
-              {formatDistanceToNow(new Date(notification.createdAt), { addSuffix: true, locale: dateFnsLocale() })}
+              {formatRelativeTime({ fromDate: new Date(notification.createdAt), locale: lang })}
```

Comment:
our function that does the same, calculates the time passed from the date to the current date